installation/resources/installer-cr.yaml.tpl

```diff
@@ -46,18 +46,20 @@ spec:
       namespace: "natss"
     - name: "core"
       namespace: "kyma-system"
+    - name: "api-gateway-migrator"
```

Comment:
That chart should be placed before the `core` chart, as we will be removing the api-controller subchart from it and the `api-gateway-migrator` tool depends on the working api-controller pod.

Reply:
The job is going to be a part of api-gateway chart.